### PR TITLE
Switch to ESM

### LINKED ISSUE #18: Switch to ESM
Many remark plugins are now using ESM. We should update this plugin as well.

### PR DESCRIPTION
**Pull request by katydecorah**
This PR updates the plugin to an ES module. This PR will update tests to use inline snapshots to make it easier to see the snapshot output.

Fixes #18

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -7,7 +7,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
+      - name: Use Node.js 14.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
```
Comment (katydecorah) [inline .github/workflows/test.yml:13]: Note: needed to specify Node 14 so that the tests would run properly.

File: babel.config.cjs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+module.exports = {
```
Comment (katydecorah) [inline babel.config.cjs:1]: This babel config supports Jest and ESM.

File: package.json [hunk 3]
```diff
@@ -43,5 +46,10 @@
   "bugs": {
     "url": "https://github.com/double-great/remark-lint-alt-text/issues"
   },
-  "homepage": "https://github.com/double-great/remark-lint-alt-text#readme"
+  "homepage": "https://github.com/double-great/remark-lint-alt-text#readme",
+  "jest": {
+    "transformIgnorePatterns": [
+      "!node_modules/"
+    ]
+  }
```
Comment (katydecorah) [inline package.json:54]: Note: needed to add this for Jest.

**Review by jsnmrs — APPROVED** [review]
Awesome!

**Pull request merged**